benchmarks/templates/benchmarks/model.html

```diff
@@ -2,17 +2,26 @@
 
 {% block content %}
 <section class="individual_model container center">
-    <h1 class="title">{{ model.name }}</h1>
-    {% if model.reference_identifier %}
-        <a href="{{ model.reference_link }}">{{ model.reference_identifier }}</a>
+    {% if model.public %}
```

Comment:
shouldn't this be `if submission_details_visible`? Otherwise, you're repeating all of this below